index.html

```diff
@@ -39,7 +39,8 @@
 		  
           };
     </script>
-<!--link rel="stylesheet" href="local.css" /-->
+<link rel="stylesheet" href="local.css">
```

Comment:
I intentionally commented out the local.css file.  We need to check what differences are made by including it.  If ok to include, we should reference the same file as other pages, so that we don't need to update in various locations.  If it needs tailoring, we should probably consider our options.